api/api.proto

```diff
@@ -2684,3 +2686,74 @@ message LocateResponse {
     // <containerid>: /var/www
     map<string, string> dockerids = 2;
 }
+
+message VolumePlacementSpec {
+  // Defines a list of rules as part of the placement spec
+  repeated VolumePlacementRule rules = 1;
+}
+
+message VolumePlacementRule {
+  // Defines the number of volume replicas affected by this rule. If not provided,
+  // rule would affect all replicas
+  // +optional
```

Comment:
When the documentation is create for the SDK website, it takes the comments as markdown, so it would show up as `replicas +optional`. To show something is optional, use `(optional)` at the beginning of the comment.